py-polars/polars/convert.py

```diff
@@ -73,7 +74,8 @@ def from_dicts(
     dicts: Sequence[dict[str, Any]], infer_schema_length: int | None = 50
 ) -> DataFrame:
     """
-    Construct a DataFrame from a sequence of dictionaries.
+    Construct a DataFrame from a sequence of dictionaries. This operation clones data,
+    unless you pass in a `dict[str, pl.Series]`.
```

Comment:
This is not correct. `from_dicts` is row oriented and will therefore always clone data to column orientation.

Reply:
OK. Fixed that!